tests/base64-test-suite.dylan

```diff
@@ -0,0 +1,30 @@
+Module: base64-test-suite
+
+define constant $text-samples
+  = #[ "Many hands make light work.",
+       "Many hands make light work" ];
```

Comment:
nit: Dylan style is to not put a space after `[` or `(`.  It's up to you whether to change it before merging. Not a big deal but I'm letting you know for future reference.